pkg/state/state_ofed.go

```diff
@@ -443,12 +452,13 @@ func (s *stateOFED) getManifestObjects(
 	renderData := &ofedManifestRenderData{
 		CrSpec: cr.Spec.OFEDDriver,
 		RuntimeSpec: &ofedRuntimeSpec{
-			runtimeSpec:    runtimeSpec{config.FromEnv().State.NetworkOperatorResourceNamespace},
-			CPUArch:        nodeAttr[nodeinfo.AttrTypeCPUArch],
-			OSName:         nodeAttr[nodeinfo.AttrTypeOSName],
-			OSVer:          nodeAttr[nodeinfo.AttrTypeOSVer],
-			MOFEDImageName: s.getMofedDriverImageName(cr, nodeAttr, reqLogger),
-			IsOpenshift:    clusterInfo.IsOpenshift(),
+			runtimeSpec:         runtimeSpec{config.FromEnv().State.NetworkOperatorResourceNamespace},
+			CPUArch:             nodeAttr[nodeinfo.AttrTypeCPUArch],
+			OSName:              nodeAttr[nodeinfo.AttrTypeOSName],
+			OSVer:               nodeAttr[nodeinfo.AttrTypeOSVer],
+			MOFEDImageName:      s.getMofedDriverImageName(cr, nodeAttr, reqLogger),
+			InitContainerConfig: s.getInitContainerConfig(cr, reqLogger, config.FromEnv().OFEDState.InitContainerImage),
```

Comment:
yeah, I was thinking about using a static provider, but I didn't find any benefits from it.
Maybe we don't need the static provider at all, because it doesn't make sense to cache env variables (we already read them only once)